

*
* Visual Studio Code Server
*
* Reminder: You may only use this software with Visual Studio family products,
* as described in the license https://aka.ms/vscode-remote/license
*

Extension host agent listening on 40163

[14:41:44] Extension host agent started.
[14:41:44] [127.0.0.1][09a66534][ManagementConnection] New connection established.
[14:41:44] [127.0.0.1][11eceda6][ExtensionHostConnection] New connection established.
[14:41:44] [127.0.0.1][11eceda6][ExtensionHostConnection] <1273> Launched Extension Host Process.
[14:46:38] [127.0.0.1][09a66534][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[14:46:39] [127.0.0.1][11eceda6][ExtensionHostConnection] <1273> Extension Host Process exited with code: 0, signal: null.
Last EH closed, waiting before shutting down
[14:46:39] Last EH closed, waiting before shutting down
Got delay-shutdown request while in shutdown timeout, delaying
[14:46:40] Got delay-shutdown request while in shutdown timeout, delaying
Cancelling previous shutdown timeout
[14:46:40] Cancelling previous shutdown timeout
[14:46:40] [127.0.0.1][a6d0c0a8][ManagementConnection] New connection established.
[14:46:40] [127.0.0.1][ba52a206][ExtensionHostConnection] New connection established.
[14:46:40] [127.0.0.1][ba52a206][ExtensionHostConnection] <1432> Launched Extension Host Process.
New EH opened, aborting shutdown
[14:51:40] New EH opened, aborting shutdown
[16:51:20] No ptyHost heartbeat after 6 seconds
rejected promise not handled within 1 second: Error: Could not find pty with id "6"
stack trace: Error: Could not find pty with id "6",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "6",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
rejected promise not handled within 1 second: Error: Could not find pty with id "6"
stack trace: Error: Could not find pty with id "6",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "6",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
[16:51:22] [127.0.0.1][a6d0c0a8][ManagementConnection] The client has reconnected.
[16:51:22] [127.0.0.1][ba52a206][ExtensionHostConnection] The client has reconnected.
[17:18:11] No ptyHost heartbeat after 6 seconds
[17:20:22] No ptyHost heartbeat after 6 seconds
[17:22:33] No ptyHost heartbeat after 6 seconds
[17:23:18] [127.0.0.1][a6d0c0a8][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/bootstrap.js:1:370)
    at process.emit (events.js:315:20)
[17:23:18] [127.0.0.1][a6d0c0a8][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
rejected promise not handled within 1 second: Error: Could not find pty with id "7"
stack trace: Error: Could not find pty with id "7",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "7",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
rejected promise not handled within 1 second: Error: Could not find pty with id "7"
stack trace: Error: Could not find pty with id "7",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "7",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
[17:23:19] [127.0.0.1][85524059][ManagementConnection] New connection established.
[17:23:19] [127.0.0.1][e573e154][ExtensionHostConnection] New connection established.
[17:23:19] [127.0.0.1][e573e154][ExtensionHostConnection] <2722> Launched Extension Host Process.
[17:23:20] [127.0.0.1][ba52a206][ExtensionHostConnection] <1432> Extension Host Process exited with code: 0, signal: null.
[17:38:11] No ptyHost heartbeat after 6 seconds
[17:38:11] [127.0.0.1][e573e154][ExtensionHostConnection] The client has reconnected.
[17:38:12] [127.0.0.1][85524059][ManagementConnection] The client has reconnected.
rejected promise not handled within 1 second: Error: Could not find pty with id "9"
stack trace: Error: Could not find pty with id "9",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "9",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
rejected promise not handled within 1 second: Error: Could not find pty with id "9"
stack trace: Error: Could not find pty with id "9",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "9",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
[17:41:28] No ptyHost heartbeat after 6 seconds
[17:45:08] No ptyHost heartbeat after 6 seconds
[17:59:38] No ptyHost heartbeat after 6 seconds
Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/bootstrap.js:1:370)
    at process.emit (events.js:315:20)
Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/bootstrap.js:1:370)
    at process.emit (events.js:315:20)
[17:59:38] [127.0.0.1][85524059][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
rejected promise not handled within 1 second: Error: Could not find pty with id "10"
stack trace: Error: Could not find pty with id "10",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "10",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
rejected promise not handled within 1 second: Error: Could not find pty with id "10"
stack trace: Error: Could not find pty with id "10",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "10",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
[17:59:40] [127.0.0.1][85524059][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[17:59:40] [127.0.0.1][12820ede][ManagementConnection] New connection established.
[17:59:40] [127.0.0.1][76251dc9][ExtensionHostConnection] New connection established.
[17:59:40] [127.0.0.1][76251dc9][ExtensionHostConnection] <3629> Launched Extension Host Process.
[17:59:57] No ptyHost heartbeat after 6 seconds
[18:00:16] No ptyHost heartbeat after 6 seconds
[18:01:00] No ptyHost heartbeat after 6 seconds
rejected promise not handled within 1 second: Error: Could not find pty with id "2"
stack trace: Error: Could not find pty with id "2",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "2",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
rejected promise not handled within 1 second: Error: Could not find pty with id "2"
stack trace: Error: Could not find pty with id "2",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "2",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
[18:01:29] No ptyHost heartbeat after 6 seconds
[18:01:33] [127.0.0.1][12820ede][ManagementConnection] The client has reconnected.
[18:01:33] [127.0.0.1][76251dc9][ExtensionHostConnection] The client has reconnected.
[18:02:47] No ptyHost heartbeat after 6 seconds
[18:05:38] [127.0.0.1][85524059][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/bootstrap.js:1:370)
    at process.emit (events.js:315:20)
Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/bootstrap.js:1:370)
    at process.emit (events.js:315:20)
Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/bootstrap.js:1:370)
    at process.emit (events.js:315:20)
[18:05:44] [127.0.0.1][12820ede][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
rejected promise not handled within 1 second: Error: Could not find pty with id "3"
stack trace: Error: Could not find pty with id "3",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "3",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
rejected promise not handled within 1 second: Error: Could not find pty with id "3"
stack trace: Error: Could not find pty with id "3",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "3",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
[18:05:45] [127.0.0.1][e573e154][ExtensionHostConnection] <2722> Extension Host Process exited with code: 0, signal: null.
[18:07:18] [127.0.0.1][12820ede][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[18:07:18] [127.0.0.1][2bb609c0][ManagementConnection] New connection established.
[18:07:18] [127.0.0.1][8b723773][ExtensionHostConnection] New connection established.
[18:07:18] [127.0.0.1][8b723773][ExtensionHostConnection] <4046> Launched Extension Host Process.
[IPC Library: Pty Host]  INFO Persistent process "12": Replaying 84 chars and 1 size events
[18:09:00] No ptyHost heartbeat after 6 seconds
rejected promise not handled within 1 second: Error: Could not find pty with id "2"
stack trace: Error: Could not find pty with id "2",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "2",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
rejected promise not handled within 1 second: Error: Could not find pty with id "2"
stack trace: Error: Could not find pty with id "2",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "2",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
[18:09:12] No ptyHost heartbeat after 6 seconds
[18:10:00] [File Watcher (node.js)] Failed to watch /home/ubuntu/contracts/LinkToken.sol for changes using fs.watch() (Error: ENOSPC: System limit for number of file watchers reached, watch '/home/ubuntu/contracts/LinkToken.sol')
[18:10:12] No ptyHost heartbeat after 6 seconds
[18:11:00] No ptyHost heartbeat after 6 seconds
[18:12:24] [127.0.0.1][12820ede][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[18:13:02] No ptyHost heartbeat after 6 seconds
[18:13:49] [127.0.0.1][76251dc9][ExtensionHostConnection] <3629> Extension Host Process exited with code: 0, signal: null.
[18:13:50] [127.0.0.1][2bb609c0][ManagementConnection] The client has reconnected.
[18:13:50] [127.0.0.1][8b723773][ExtensionHostConnection] The client has reconnected.
[18:27:37] No ptyHost heartbeat after 6 seconds
rejected promise not handled within 1 second: Error: Could not find pty with id "3"
stack trace: Error: Could not find pty with id "3",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "3",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
rejected promise not handled within 1 second: Error: Could not find pty with id "3"
stack trace: Error: Could not find pty with id "3",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "3",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
[18:29:49] [127.0.0.1][2bb609c0][ManagementConnection] The client has reconnected.
[18:29:49] [127.0.0.1][8b723773][ExtensionHostConnection] The client has reconnected.
[18:30:40] No ptyHost heartbeat after 6 seconds
rejected promise not handled within 1 second: Error: Could not find pty with id "4"
stack trace: Error: Could not find pty with id "4",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "4",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
rejected promise not handled within 1 second: Error: Could not find pty with id "4"
stack trace: Error: Could not find pty with id "4",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "4",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
[18:31:33] [127.0.0.1][2bb609c0][ManagementConnection] The client has reconnected.
[18:31:33] [127.0.0.1][8b723773][ExtensionHostConnection] The client has reconnected.
[19:04:15] No ptyHost heartbeat after 6 seconds
Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/bootstrap.js:1:370)
    at process.emit (events.js:315:20)
Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/bootstrap.js:1:370)
    at process.emit (events.js:315:20)
Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/bootstrap.js:1:370)
    at process.emit (events.js:315:20)
Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/bootstrap.js:1:370)
    at process.emit (events.js:315:20)
[19:04:53] [127.0.0.1][2bb609c0][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[19:04:54] [127.0.0.1][2bb609c0][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[19:04:54] [127.0.0.1][d80c8004][ExtensionHostConnection] New connection established.
[19:04:54] [127.0.0.1][35920420][ManagementConnection] New connection established.
[19:04:54] [127.0.0.1][d80c8004][ExtensionHostConnection] <5462> Launched Extension Host Process.
rejected promise not handled within 1 second: Error: Could not find pty with id "5"
stack trace: Error: Could not find pty with id "5",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "5",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
rejected promise not handled within 1 second: Error: Could not find pty with id "5"
stack trace: Error: Could not find pty with id "5",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "5",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
[IPC Library: Pty Host]  INFO Persistent process "12": Replaying 2684 chars and 4 size events
[IPC Library: Pty Host]  INFO Persistent process "15": Replaying 4220 chars and 3 size events
[19:05:26] No ptyHost heartbeat after 6 seconds
[19:06:11] No ptyHost heartbeat after 6 seconds
[19:07:38] No ptyHost heartbeat after 6 seconds
[19:10:15] [127.0.0.1][2bb609c0][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[19:11:40] No ptyHost heartbeat after 6 seconds
rejected promise not handled within 1 second: Error: Could not find pty with id "2"
stack trace: Error: Could not find pty with id "2",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "2",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
[19:11:45] [127.0.0.1][8b723773][ExtensionHostConnection] <4046> Extension Host Process exited with code: 0, signal: null.
rejected promise not handled within 1 second: Error: Could not find pty with id "2"
stack trace: Error: Could not find pty with id "2",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "2",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
[19:11:49] [127.0.0.1][35920420][ManagementConnection] The client has reconnected.
[19:11:49] [127.0.0.1][d80c8004][ExtensionHostConnection] The client has reconnected.
rejected promise not handled within 1 second: Error: Could not find pty with id "3"
stack trace: Error: Could not find pty with id "3",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "3",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateTitle (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46404),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
rejected promise not handled within 1 second: Error: Could not find pty with id "3"
stack trace: Error: Could not find pty with id "3",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "3",    at y._throwIfNoPty (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:49364),    at y.updateIcon (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:12:46464),    at Object.call (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:11:8338),    at y.onPromise (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49795),    at y.onRawMessage (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:49154),    at /home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:10:48407,    at T.fire (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:8:1726),    at process.U (/home/ubuntu/.vscode-server/bin/c3f126316369cd610563c75b1b1725e0679adfb3/out/vs/platform/terminal/node/ptyHostMain.js:6:31321),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
